CMakeLists.txt

```diff
@@ -10,9 +10,14 @@ idf_component_register(
          "src/crypto/refc/x25519.c"
          "src/esp_wireguard.c"
          "src/nacl/crypto_scalarmult/curve25519/ref/smult.c"
+         "src/derp.c"
```

Comment:
Maybe it is better to exclude derp from this PR and include it in another with all the DERP stuff ..?